logger.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+// ServiceLogger represents a logger with logging level prefixes for a specific service.
+type ServiceLogger struct {
+	ServiceName string
+	Debug       *log.Logger
+	Info        *log.Logger
+	Fatal       *log.Logger
+	Error       *log.Logger
+	*log.Logger
+}
+
+// NewServiceLogger returns a logger with designated logging levels for a particular service.
+func NewServiceLogger(serviceName string, debug bool) ServiceLogger {
```

Comment:
in the true spirit of logging levels, we need to pass in the logging level and set any level below that to be similar to line 30 where it's discarded

Reply:
I will reference this https://dave.cheney.net/2015/11/05/lets-talk-about-logging 😉 
>log.Info should simply write that line to the log output. There should not be an option to turn it off as the user should only be told things which are useful for them.
but don't mind having the extra feature

Reply:
Agree on logging levels -- we should have the ability to set it for different environments to control the chattiness of the applications.